### DISCUSSION TITLE
Regarding the relations between the enemies of the deeper dark, and sculk magpie origin.

### DISCUSSION THREAD

**Discussion by Timtee** [KyaniteMods/DeeperAndDarker#175] [Question] [answered]
Seeing as by default the warden is entirely friendly to the sculk magpie origin that can be gotten as an additional origin to the Origins mod... I have to wonder if the enemies of the Deeper dark would also be friendly towards the magpie, or aggressive. Mostly just because of curiosity primarily.

**Comment by nitrodynamite18** [comment, marked as answer]
Other Sculk mobs would also be passive toward the magpie, yes.
